rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/CreateQueueDirectives.scala

```diff
@@ -13,6 +13,7 @@ import CreateQueueDirectives._
 
 trait CreateQueueDirectives { this: ElasticMQDirectives with QueueURLModule with AttributesModule with SQSLimitsModule =>
 
+  // DeadLettersQueue currently not supported via http call due to current API limitations. Consider moving to JSON?
```

Comment:
didn't get it, RedrivePolicy expects a string with json like `"{\"maxReceiveCount\":\"5\", \"deadLetterTargetArn\":\"dead_letters_queue1\"}"` ?